corehq/apps/cloudcare/static/cloudcare/js/formplayer/apps/menus/menu_controller.js

```diff
@@ -48,15 +48,27 @@ FormplayerFrontend.module("SessionNavigate.MenuList", function (MenuList, Formpl
             if (menuListView) {
                 FormplayerFrontend.regions.main.show(menuListView.render());
             }
+            if (menuResponse.persistentCaseTile) {
+                MenuList.Controller.showPersistentCaseTile(menuResponse.persistentCaseTile);
+            } else {
+                $("#persistent-case-tile").html("");
+            }
 
             if (menuResponse.breadcrumbs) {
                 MenuList.Controller.showBreadcrumbs(menuResponse.breadcrumbs);
+            } else {
+                $("#breadcrumb-region").html("");
```

Comment:
less fragile would be:  `FormplayerFrontend.regions.breadcrumb.empty();`. in case we ever change the id name (same with above)